package.json

```diff
@@ -1,6 +1,6 @@
 {
     "name": "@nationalarchives/file-information",
-    "version": "1.0.0",
+    "version": "1.0.1",
```

Comment:
I see we've hit 1.0 already. 😂

Reply:
Npm doing its own thing with defaults again.

Reply:
Have you published to npm yet? Can we reset it to 0.x or is it too late?

Reply:
Yeah, I published it and based on the npm guidelines, we shouldn't anyway 
https://docs.npmjs.com/about-semantic-versioning

Reply:
Heh. I can see why they don't like 0.x because of the number of projects that just get stuck there, but it does make it look like our library is ready to use. 🙈